applications/CableNetApplication/custom_elements/sliding_cable_element_3D.cpp

```diff
@@ -338,9 +339,15 @@ Vector SlidingCableElement3D::GetInternalForces()
   Values.SetStressVector(temp_stress);
   mpConstitutiveLaw->CalculateMaterialResponse(Values,ConstitutiveLaw::StressMeasure_PK2);
 
-
+ 
   double total_internal_force = (temp_stress[0]+prestress) * area * current_length / ref_length;
 
+  mIsCompressed = false;
+  if ((total_internal_force < 0.00)&&(std::abs(current_length-ref_length)>numerical_limit)) {
```

Comment:
Do you knwo what is more probable to be false?, because the one more probable to be false should go first

Reply:
OK, good to know. It is already the right way around